### ISSUE TITLE
Leaves invalid statement on erased unused imports from "import * as x from 'x'" statements

### ISSUE THREAD

**Issue by andfaulkner** [unlight/sublime-import-helper#56]
For example, if I have this import statement, where someLib isn't used in the current file:
```
import * as someLib from 'some-lib';
```
...when I run "Remove unused imports", it outputs:
```
import * as from 'some-lib';
```

I'd expect it to erase the entire import statement instead.

**Comment by unlight** [comment]
Fixed in v1.8.2

**Comment by andfaulkner** [comment]
Thanks @unlight, much appreciated :)
